cmd/tools/genrpcwrappers/extra.go

```diff
@@ -28,23 +28,36 @@ var (
 		"SyncWorkflowState":                   true,
 
 		// workflowservice
-		"DescribeDeployment":             true,
-		"ExecuteMultiOperation":          true,
-		"GetCurrentDeployment":           true,
-		"GetDeploymentReachability":      true,
-		"GetWorkerVersioningRules":       true,
-		"ListDeployments":                true,
-		"PauseActivityById":              true,
-		"PollNexusTaskQueue":             true,
-		"ResetActivityById":              true,
-		"RespondNexusTaskCompleted":      true,
-		"RespondNexusTaskFailed":         true,
-		"SetCurrentDeployment":           true,
-		"ShutdownWorker":                 true,
-		"UnpauseActivityById":            true,
-		"UpdateActivityOptionsById":      true,
-		"UpdateWorkerVersioningRules":    true,
-		"UpdateWorkflowExecutionOptions": true,
+
+		"DeleteWorkerDeployment":                true,
+		"DeleteWorkerDeploymentVersion":         true,
+		"DescribeDeployment":                    true,
+		"DescribeWorkerDeployment":              true,
+		"DescribeWorkerDeploymentVersion":       true,
+		"ExecuteMultiOperation":                 true,
+		"GetCurrentDeployment":                  true,
+		"GetDeploymentReachability":             true,
+		"GetWorkerVersioningRules":              true,
+		"ListDeployments":                       true,
+		"ListWorkerDeployments":                 true,
+		"PauseActivity":                         true,
+		"PauseActivityById":                     true,
+		"PollNexusTaskQueue":                    true,
+		"ResetActivity":                         true,
+		"ResetActivityById":                     true,
+		"RespondNexusTaskCompleted":             true,
+		"RespondNexusTaskFailed":                true,
+		"SetCurrentDeployment":                  true,
+		"SetWorkerDeploymentCurrentVersion":     true,
+		"SetWorkerDeploymentRampingVersion":     true,
+		"ShutdownWorker":                        true,
+		"UnpauseActivity":                       true,
+		"UnpauseActivityById":                   true,
+		"UpdateActivityOptions":                 true,
+		"UpdateActivityOptionsById":             true,
+		"UpdateWorkerDeploymentVersionMetadata": true,
+		"UpdateWorkerVersioningRules":           true,
+		"UpdateWorkflowExecutionOptions":        true,
```

Comment:
ignore new apis for the utf8 repair stuff (needs old protos)

Reply:
we might need some steps on how to update these files for future.